packages/web-pkg/src/composables/piniaStores/config/config.ts

```diff
@@ -79,7 +81,10 @@ export const useConfigStore = defineStore('config', () => {
     theme.value = data.theme
 
     if (data.options) {
-      options.value = merge({ ...defaultOptions }, data.options)
+      // routing will not be loaded from config, but set based on cernFeatures option
+      options.value = merge({ ...defaultOptions }, omit(data.options, ['routing']))
```

Comment:
maybe the omit is not needed, just a security layer, so that routing will not be loaded at all via config